src/raftstore/store/peer.rs

```diff
@@ -597,34 +598,42 @@ impl Peer {
         pending_peers
     }
 
-    pub fn check_stale_state(&mut self, d: Duration) -> StaleState {
+    pub fn check_stale_state(&mut self) -> StaleState {
         // Updates the `leader_missing_time` according to the current state.
         if self.leader_id() == raft::INVALID_ID {
             if self.leader_missing_time.is_none() {
                 self.leader_missing_time = Some(Instant::now())
             }
         } else if self.is_initialized() {
-            // A peer is considered as in the leader missing state if it's uninitialized or
```

Comment:
Why move the comment? So why is `leader_missing_time` set to None when it is initialized?